# This is a template parameter file which can be copied and modified for
# a specific implementation
/**:
  ros__parameters:
    cycle_time_ms: 10
    # Only one of the four control command topics need be specified
    # "raw", "basic", "ackermann", or "high_level"
    control_command: "raw"
    state_machine:
      gear_shift_velocity_threshold_mps: 0.5
      acceleration_limits:
        min: -3.0
        max: 3.0
        threshold: 1.0
      front_steer_limits:
        min: -0.331
        max: 0.331
        threshold: 0.3
      time_step_ms: 100
      timeout_acceleration_mps2: 3.0
      state_transition_timeout_ms: 3000
      gear_shift_accel_deadzone_mps2: 0.5
    features: ["headlights", "horn", "wipers"]
